scripts/run-integration-tests-source.sh

```diff
@@ -220,7 +220,7 @@ rm -Rfv \
 
 # --- Running Mock Blockchain --- {{{
 log "Running hardhat local node"
-HOPR_ENVIRONMENT_ID="hardhat-localhost" yarn workspace @hoprnet/hopr-ethereum hardhat node \
+HOPR_ENVIRONMENT_ID="hardhat-localhost" TS_NODE_PROJECT=${mydir}/../packages/ethereum/tsconfig.hardhat.json yarn workspace @hoprnet/hopr-ethereum exec "hardhat node" \
```

Comment:
What does this extra env var do?

Reply:
Make sure that ts-node picks the right tsconfig file